File: postbuild.py
import os
import shutil

UNUSED_FILES = [
    "main.js",
]
UNUSED_DIRS = [
    "_snowpack",
    "test",
]

def main():
    for uf in UNUSED_FILES:
        os.remove(f"build/{uf}")

    for ud in UNUSED_DIRS:
        shutil.rmtree(f"build/{ud}")

if __name__ == "__main__":
    main()

File: postbuild.js
const fs = require("fs");
const path = require("path");

const UNUSED_FILES = [
  "main.js",
];
const UNUSED_DIRS = [
  "_snowpack",
  "test",
];

for (const uFile of UNUSED_FILES) {
  fs.unlinkSync(path.join("build", uFile), { recursive: true });
}

for (const uDir of UNUSED_DIRS) {
  fs.rmdirSync(path.join("build", uDir), { recursive: true });
}
